wpeverest/user-registration
# UR-1490 Dev - Compatibility with Repeater Fields Addon

### All Submissions:






### Changes proposed in this Pull Request:
This PR provides compatibility with Repeater Fields Addon.

### How to test the changes in this Pull Request:

1. Without activating Repeater Fields Addon, test this PR to see if anything in our plugins breaks.
2. Activate Repeaters Field Addon and Pro and test if anything breaks.

### Types of changes:




### Other information:




### Changelog entry

> Dev - Compatibility with Repeater Fields Addon
